src/main/java/seedu/address/logic/commands/HelpConsultationCommand.java

```diff
@@ -0,0 +1,16 @@
+package seedu.address.logic.commands;
```

Comment:
Hi @nerdyboy98 , do we really need these sub-classes? I feel we may statically save string messages inside the parent class, and differentiate them in the initializer? You may decide on this.

Reply:
I tried doing that but the issue is with the execute method, which only returns 1 String message. Im not sure of how to enable if-else statement to work with that thats why i created these subclasses D: